code/obj/item/organs/organ_parent.dm

```diff
@@ -339,3 +352,15 @@
 			for (var/abil in src.organ_abilities)
 				src.add_ability(A, abil)
 		src.broken = 0
+
+	// This proc converts a hex color value ("#420CAB") to an RGB list
+	// Clamps each of the RGB values between 50 and 190
+	// So we cant make our organs look *too* stupid through character settings
+	proc/organ_fix_colors(var/hex)
+		var/list/L = hex_to_rgb_list(hex)
+		for (var/i in L)
+			L[i] = min(L[i], 190)
+			L[i] = max(L[i], 50)
+		if (L.len == 3)
+			return rgb(L["r"], L["g"], L["b"])
+		return rgb(22, 210, 22)
```

Comment:
isn't there a proc to do this already?

Reply:
Not only is there a proc to do this already, but its copy-pasted in no less than 4 places. By me. Who thought this was a good idea for some reason. I'm just going to move it to color.dm.